Java/Crypto/src/main/java/com/breadwallet/crypto/WalletManager.java

```diff
@@ -9,12 +9,16 @@
  */
 package com.breadwallet.crypto;
 
+import com.breadwallet.crypto.errors.FeeEstimationError;
+import com.breadwallet.crypto.utility.CompletionHandler;
 import com.google.common.base.Optional;
 
 import java.util.List;
 
 public interface WalletManager {
 
+    void createSweeper(byte[] forPrivateKeyUt8, CompletionHandler<WalletSweeper, WalletSweeper.WalletSweeperError> completion);
+
```

Comment:
Can't the first argument here be `Key` (w/ the expectation/check that it is a private key)?

Reply:
Sure, easy enough to change.